### PR TITLE
Addition of ARC header

### PR DESCRIPTION
**Pull request by joinimran**
Addition of ARC header in the list.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/822

### CODE REVIEW

**Comment by joinimran** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/anti-spam-message-headers.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ After accessing the message header information, search for **X-Forefront-Antispa
 |SFV:NSPM|The message was marked as non-spam and was sent to the intended recipients.|
 |H: \[helostring\]|The HELO or EHLO string of the connecting mail server.|
 |PTR: \[ReverseDNS\]|The PTR record, or pointer record, of the sending IP address, also known as the reverse DNS address.|
+|ARC:|The ARC protocol has three headers as: <br/>AAR: Records the content of Authentication results header from DMARC. <br/>AMS: This includes cryptographic signatures of the message <br/>AS: Includes cryptographic signature of the message headers. This header contains a tag of chain validation called "cv=" which includes the outcome of the chain validation as **none, pass or fail**.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/anti-spam-message-headers.md]:
```suggestion
|ARC:|The ARC protocol has three headers, as follows: <br/>AAR: Records the content of the Authentication results header from DMARC. <br/>AMS: This header includes cryptographic signatures of the message. <br/>AS: Includes cryptographic signatures of the message headers. This header contains a tag of a chain validation called "cv=", which includes the outcome of the chain validation as **none**, **pass**, or **fail**.|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review, suggestions.

**Comment by joinimran** [comment]
@mypil suggestions applied. Ready to go live.

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #822

Thank you.

cc: @kenwith

**Pull request merged**